nntrainer/tensor/manager.cpp

```diff
@@ -177,23 +177,23 @@ void Manager::trackWeights(std::vector<Weight> &ws) {
 }
 
 Manager::AllocFunc Manager::getAllocFunc(bool is_weight) {
-  AllocFunc allocate_none = [](const TensorDim &dim, size_t) {
+  AllocFunc allocate_none = [](const TensorDim &dim, unsigned int) {
```

Comment:
any reason for the change?

Reply:
Tensor uses unsigned int, and it causes `unsigned int` vs `long unsigned int` comparison warnings at a lot of places.

Reply:
As far as I remember, size_t can effectively check max overflow but unsigned int is not guaranteed as max supported size is different. I guess not a big deal for now

Reply:
Yes, size_t is better, but it requires repo wide updates, which is beyond the scope of this PR.